xarray/tests/test_save_mfdataset_pass_kwargs.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+import xarray as xr
+
+
+@pytest.mark.parametrize("test_type", ["baseline", "encoding_and_unlimited_dims"])
+def test_save_mfdataset_pass_kwargs(test_type):
```

Comment:
Could we move this test in the same test file as the rest of `save_mfdataset`  tests? https://github.com/pydata/xarray/blob/aa1d1d19b822897399c8ed2cf346afbac71f45b3/xarray/tests/test_backends.py#L3672

Reply:
sure thing.  I'm not sure what the protocol for squashing commits is here: should I push a separate commit, or should I squash it with the original?

Reply:
we squash commits when merging into main. So, i wouldn't worry about squashing commits in this PR.

Reply:
okay, got it, thanks!